ibm/service/cos/resource_ibm_cos_bucket_object.go

```diff
@@ -615,3 +723,18 @@ func deleteCOSObjectVersion(conn *s3.S3, b, k, v string, force bool) error {
 
 	return err
 }
+func parseDate(v string) *time.Time {
```

Comment:
Can you move this utility function to flex.go file

Reply:
For all the resource files we have the utility functions in the same file , hence I also added there itself.